pytests/repo/setup-repo.sh

```diff
@@ -53,9 +53,8 @@ EOF
 
 echo building packages
 rpmbuild  --define "_topdir ${BUILD_PATH}" \
-	  --define "__gpg_sign_cmd %%{__gpg} gpg --batch" \
```

Comment:
I think we need to do this based on version of Photon/rpmbuild used.

Reply:
Fedora-33 also moved to rpm-4.16.x, I think we are using fedora-32 in our ci, can you check with fedora:latest please?

Reply:
> I think we need to do this based on version of Photon/rpmbuild used.

I tested with Photon 3.0, works fine. Photon 2.0 has rpmsign too.

Reply:
> Fedora-33 also moved to rpm-4.16.x, I think we are using fedora-32 in our ci, can you check with fedora:latest please?

Just tested with `fedora:latest` too, it works.